ocaml/xapi/helpers.ml

```diff
@@ -63,10 +63,16 @@ let get_pool ~__context = List.hd (Db.Pool.get_all ~__context)
 let get_master ~__context =
   Db.Pool.get_master ~__context ~self:(get_pool ~__context)
 
-let get_management_iface_is_connected ~__context =
+let get_iface_is_connected ~__context iface =
   let dbg = Context.string_of_task __context in
-  Net.Bridge.get_physical_interfaces dbg (Xapi_inventory.lookup Xapi_inventory._management_interface)
-  |> List.exists (fun name -> Net.Interface.is_connected dbg name)
+  Net.Bridge.get_physical_interfaces dbg iface
+  |> List.exists (fun name ->
+    debug "Checking interface %s" name;
```

Comment:
Let's drop this debug line. It doesn't add much and we are already logging too much.